app/models/paper.rb

```diff
@@ -2,4 +2,7 @@ class Paper < ApplicationRecord
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references
+
+  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
```

Comment:
I believe when I read the tutorial, this validation was needed here for nested attributes.  I'll double cheeck, but its meant to not create blank authors.  Ie, I have 5 author fields and if I enter 4 names, only 4 will be created.